### ISSUE TITLE
The color scheme is not consistent with VS Code. import Purple/pink color is missing.

### ISSUE THREAD

**Issue by twilingway** [samdark/intellij-visual-studio-code-dark-plus#39]
### What the issue is?
if i set Keywords in WS function, const var let also change color
### Screenshots
https://yadi.sk/i/Cpx2IfUBuJB9Og
also component color wrong
https://yadi.sk/i/JRqC2BxpHUlRjw
### Additional info

| Q                | A
| ---------------- | ---
| Plugin Version   | 2.3
| IDE and version  | 2020.1
| Operating system |

**Comment by adrien-cuisse** [comment]
The problem is that PhpStorm puts all keywords in the same bag.

Here's a piece of code using VSCode, you can see conditions and return getting another highlight 
![image](https://user-images.githubusercontent.com/60260640/111916389-04bc6c80-8a7b-11eb-970e-e70a9bdeb8b9.png)

The same piece of code on PhpStop
![image](https://user-images.githubusercontent.com/60260640/111916587-ef940d80-8a7b-11eb-9f37-c9d2e11b3095.png)

You can notice the isset having wrong color too (keyword for PhpStorm)

**Comment by samdark** [comment]
Yeah. Can not fix it until PhpStorm will see these as different types.

**Comment by dinbtechit** [comment]
Perhaps it's a good idea to have a separate plugin for PhpStorm? Having `import` highlighted as purple/pink would be really cool :). Thoughts?

**Comment by samdark** [comment]
@dinbtechit well, extra tokenizing could theoretically be done in this plugin but I doubt I currently have enough time to implement it.

**Comment by dinbtechit** [comment]
@samdark - I have time to work on this but having a hard time wrapping my head around how tokenizing works. Can you give me a basic example of how to create tokenization in this plugin?

Like you said,  `class`, 'implementation`, etc and `import` are in the same bucket `keyword`.  Looks like there is no way to define a sub-bucket under `keyword`. Have you done any research on that? Is there any way we can separate them out? Sadly, JetBrain's documentation is very limited on this. 


![image](https://user-images.githubusercontent.com/17984781/131233595-2c69d693-17e3-4ad8-bfc8-3f6b89615db9.png)

**Comment by samdark** [comment]
@dinbtechit I'm not aware of it :(
